xbmc/guilib/FFmpegImage.cpp

```diff
@@ -266,7 +291,7 @@ bool CFFmpegImage::Decode(unsigned char * const pixels, unsigned int width, unsi
     width, height, AV_PIX_FMT_RGB32, SWS_BICUBIC, NULL, NULL, NULL);
 
   sws_scale(context, m_pFrame->data, m_pFrame->linesize, 0, m_height,
-    pictureRGB->data, pictureRGB->linesize);
+      pictureRGB->data, pictureRGB->linesize);
```

Comment:
Just in case you change something else as well: either pad it to context or leave it as is